### PR TITLE
Feat: camera reel storage services

### PR DESCRIPTION
**Pull request by popuz**
## What does this PR change?

Adds infrastructure layer for connecting with the Camera Reel backend to Get/Delete/Upload screenshots with their metadata. Currently working only GET request (others are Unauthorized and will be solved it on the back end side)

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by lorenzo-ranciaffi — APPROVED** [review]

File: Explorer/Assets/DCL/WebRequests/GenericDownloadHandlerUtils.cs [hunk 1]
```diff
@@ -20,28 +20,6 @@ public static class GenericDownloadHandlerUtils
     {
         public delegate Exception CreateExceptionOnParseFail(Exception exception, string text);
 
-        public static Adapter<GenericPostRequest, GenericPostArguments> SignedFetchPostAsync(
```
Comment (pravusjif) [inline Explorer/Assets/DCL/WebRequests/GenericDownloadHandlerUtils.cs:23]: is it OK to remove this one?

File: Explorer/DCL.CameraReelStorageService.csproj.DotSettings
Comment (pravusjif) [inline Explorer/DCL.CameraReelStorageService.csproj.DotSettings:1]: is this file needed?

**Review by pravusjif — APPROVED** [review]
LGTM, in the future we'll see if the string allocations are a problem or not

File: Explorer/Assets/DCL/WebRequests/GenericDownloadHandlerUtils.cs (hunk 1 above)
Comment (popuz) [reply Explorer/Assets/DCL/WebRequests/GenericDownloadHandlerUtils.cs:23]: I just moved it inside another extension file for all SignedFetches

File: Explorer/DCL.CameraReelStorageService.csproj.DotSettings
Comment (popuz) [reply Explorer/DCL.CameraReelStorageService.csproj.DotSettings:1]: yep, it disables namespaces matches with subfolders. We generally do it, no harm
